client.go

```diff
@@ -105,9 +116,20 @@ func NewFileClient(keyID string) (Client, error) {
 	return c, nil
 }
 
+//Creates a mockKeyVersion to be used for testing
```

Comment:
Use the same comment syntax as jon recommended above.

Reply:
Updated.